recognizers/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.sonarsource.analyzer-commons</groupId>
+    <artifactId>sonar-analyzer-commons-parent</artifactId>
+    <version>1.5-SNAPSHOT</version>
```

Comment:
We will have to increment that if we don't want to merge this PR before the release of 1.5.

Reply:
Oh, good point!